src/client/models/externalVariables.js

```diff
@@ -0,0 +1,15 @@
+module.exports.originalPoints = [[0, 280],[200, 350], [350, 150], [500, 250],[800, 250],
+[800, 500],[0, 500],[0, 280]];
+
+module.exports.WIDTH = 800;
```

Comment:
this style is more server side one. in FE we do just

export const WIDTH = 800;

or 

export default {
 WIDTH: 800
}